Docs grammar and typos check (#6064)
vuestorefront/vue-storefront: packages/core/docs/contributing/server-side-rendering.md

# Server-side rendering

> In Vue Storefront, we rely on Nuxt.js as a server-side rendering provider, and we highly recommend this as well.
> Although Nuxt.js has everything we need regarding SSR, we had to provide a bit more to this implementation.

Reviewer: ```suggestion
In Vue Storefront, we rely on Nuxt.js as a server-side rendering provider. Although it is a powerful framework, we needed to extend it to meet our needs.
```

As we still wait for full integration of Vue 3 and Nuxt.js, and the current SSR based on

> composition-api didn't cover our needs, we had to temporarily sort it out on our own.

Reviewer: Would you mind deleting this sentence? I'm sure we will not remember to do it once we upgrade to Vue 3 / Nuxt 3.
